db/commitlog/commitlog_replayer.cc

```diff
@@ -12,6 +12,7 @@
 #include <vector>
 #include <algorithm>
 #include <unordered_map>
+#include <ranges>
```

Comment:
nit: typo in commit log message: s/Mimizes/Minimizes/

Reply:
fixed